atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/asts/ShardedSweepableBucketRetrieverTest.java

```diff
@@ -136,8 +129,8 @@ public void backoffCanBeInterrupted() throws InterruptedException {
         future.cancel(true);
         executorService.shutdown();
 
-        // Even though the backoff in 10s, interrupting the task should make it finish much faster.
-        Awaitility.await().atMost(Duration.ofMillis(200)).untilAsserted(() -> assertThat(
+        // Even though the backoff is up to 10 minutes, interrupting the task should make it finish much faster.
+        Awaitility.await().atMost(Duration.ofSeconds(1)).untilAsserted(() -> assertThat(
```

Comment:
defaultSleeperCanBeInterrupted will involve randomness once again, as we're not pinning the backoff value (it's back to being controlled by TLR) - and so it could spuriously pass. It's a 0.16% chance for that to happen ((1000 / 600000) * 100) - I can reduce the odds even further by bumping the maxBackoff, but in cases where it fails legitimately, CI will just spin.